lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
oh okay that makes sense! I'm having a difficult time seeing where this would be changed in Rogue. Does it make sense to create a new file `PaginatorForPhoenixAshesGallery` that extends `IlluminatePaginatorAdapter` and call that [here](https://github.com/DoSomething/rogue/blame/master/app/Http/Controllers/Api/ApiController.php#L119)? Then in `PaginatorForPhoenixAshesGallery` rewrite `paginator` function to add `next_uri` instead of `next`?